.github/workflows/ci-testing.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           pip list
 
       - name: Tests
-        timeout-minutes: 10
+        timeout-minutes: 30
```

Comment:
Let's switch it back since we don't want to run tests for 30 mins. 

```suggestion
        timeout-minutes: 10
```

Reply:
It seems CI is stuck since this commit https://github.com/Lightning-AI/LitServe/commit/2bbed42b69d4906ea34f7ad0248bc435a20bdad3

Maybe due to version 3.11 of python for a reason i ignore.

Reply:
yes, since it's only specific to Python 3.11 on Windows latest, it probably means that something is not working as expected.